frontend/src/components/Notification/Notification.tsx

```diff
@@ -48,29 +41,18 @@ const Notification = () => {
 
   const mapMessages = (msgs: Message[]) =>
     msgs.map(({ time, title, body }, i) => (
-      <div key={i} className={styles.body}>
-        <div className={styles.user}>
-          <div className={styles.avatar}>
-            <span className={styles.initials}>C</span>
-          </div>
-        </div>
-        <div className={styles.msg}>
-          <p className={styles.date}>{moment(time).format('MMMM Do')}</p>
-          <p>{body}</p>
-        </div>
-        <div className={styles.link}>View</div>
-      </div>
+      <DisplayMessage key={i} {...{ time, title, body }} />
     ));
 
-  const element = document.getElementsByClassName(
-    'notification_bell__HML54'
-  )[0]!;
-  element.removeAttribute('aria-describedby');
+  useEffect(() => {
+    const element = document.getElementById('popup');
+    element?.removeAttribute('aria-describedby');
+  }, []);
 
   return (
     <Popup
       trigger={
-        <button className={styles.bell}>
+        <button id="popup" className={styles.bell}>
```

Comment:
I've already commented on this, but we should not be hardcoding id's within components. Maybe we can fix this after we migrate to modern React (18) where you get the `useId` hook natively.